Toric/Mathlib/Algebra/Order/Nonneg/Module.lean

```diff
@@ -0,0 +1,31 @@
+import Mathlib.Algebra.Order.Nonneg.Module
+import Mathlib.RingTheory.Finiteness.Basic
+
+namespace Nonneg
+
+section Ring
+
+variable (𝕜 E : Type*) [Ring 𝕜] [LinearOrder 𝕜] [IsOrderedRing 𝕜] [AddCommMonoid E] [Module 𝕜 E]
+
+/-- A linearly ordered ring is finitely generated as a module over the non-negative scalars. -/
+instance isFiniteOver : Module.Finite {c : 𝕜 // 0 ≤ c} 𝕜 := by
```

Comment:
Can you make this `private` since it's subsumed by the next instance?

Reply:
made it private